app/src/main/java/tech/relaycorp/gateway/data/preference/PublicGatewayPreferences.kt

```diff
@@ -30,8 +34,11 @@ class PublicGatewayPreferences
 
     fun getCertificate() = { certificate }.toFlow()
         .map {
-            if (it.isEmpty()) throw RuntimeException("Public Gateway Certificate not set")
-            val certificateBytes = it.toByteArray(Charset.defaultCharset())
+            val certificateBytes = if (it.isEmpty()) {
+                resources.openRawResource(R.raw.public_gateway_cert).readBytesAndClose()
+            } else {
```

Comment:
FYI: This is OK for now, but we'll probably have to set all the public gateway data upfront, when the app is started for the first time.